app/Helpers/database/forum.php

```diff
@@ -152,10 +152,6 @@ function setLatestCommentInForumTopic(int $topicID, int $commentID): bool
 
 function editTopicComment(int $commentID, string $newPayload): bool
 {
-    $newPayload = str_replace("'", "''", $newPayload);
```

Comment:
Seems like this like is still required. Maybe the query should use a binding?

Reply:
1 moment, it's simple enough I can just convert it to Eloquent ORM.

Reply:
Resolved in latest